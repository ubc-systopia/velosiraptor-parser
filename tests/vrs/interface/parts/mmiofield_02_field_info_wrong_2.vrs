mmio foo [ base, 8 ] {Layout {}}