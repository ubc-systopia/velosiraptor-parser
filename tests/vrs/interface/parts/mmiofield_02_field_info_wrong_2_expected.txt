error: expected `,`, found `]`
     --> tests/vrs/interface/parts/mmiofield_02_field_info_wrong_2.vrs:1:20
      |
    1 |  mmio foo [ base, 8 ] {Layout {}}
      |                     ^ help: add `,` here.
